functions/PushNotificationTutorConfirmation.js

```diff
@@ -0,0 +1,111 @@
+const {onDocumentUpdated} = require("firebase-functions/v2/firestore");
+const {setGlobalOptions} = require("firebase-functions/v2");
+const {initializeApp} = require("firebase-admin/app");
+const {getFirestore} = require("firebase-admin/firestore");
+const admin = require("firebase-admin");
+
+initializeApp();
+
+setGlobalOptions({
+  maxInstances: 10,
+  timeoutSeconds: 60,
+  memory: "256MiB"});
+
+exports.pushNotificationTutorLessonConfirmation = onDocumentUpdated(
+    "lessons/{lessonId}",
+    async (event) => {
+      const before = event.data.before.data(); // Data before the update
+      const after = event.data.after.data(); // Data after the update
+      const lessonId = event.params.lessonId;
+      const db = getFirestore();
+
+      // Check if the status field changed
+      if (before.status !== after.status) {
+        console.log(`Lesson ${lessonId} status changed from ${before.status}
```

Comment:
Please use a `const logger = require("firebase-functions/logger");` as Alexis did in index.js as we will probably have some erros in the future and we want to use the same logger for all errors related to .js functions in Firebase